client/src/partials/patient_invoice/patientInvoice.js

```diff
@@ -155,6 +159,22 @@ function PatientInvoiceController(Patients, PatientInvoices, PatientInvoiceForm,
 
     if (angular.isDefined(vm.services) && vm.services.length) {
       vm.Invoice.setService(vm.services[SERVICE_INDEX]);
+      vm.selectedService = vm.services[SERVICE_INDEX];
+    }
+  }
+
+  function setService (service_id) {
+
+    Services.read(service_id)
+      .then(function (service) {
+        vm.selectedService = service;
+        updateDescription();
+      });
+  }
+
+  function updateDescription() {
+    if(vm.Invoice.recipient && vm.selectedService){
+      vm.Invoice.details.description = [vm.descriptionPrefix, vm.Invoice.recipient.display_name, vm.selectedService.name].join(' ');
```

Comment:
Does White spaces are good separators for the description ? 
Ex. `invoice Dedrick Kitamuka Medecine Interne` 

What do you think about `/`, 
Ex. `invoice/Dedrick Kitamuka/Medecine Interne` 
In this case, we can easily distinguish the patient name and the service.